### DISCUSSION TITLE
How do I display a message when no consent is given?

### DISCUSSION THREAD

**Discussion by morja** [orestbida/cookieconsent#734] [Q&A]
I thougth I could do something like
```
<script
        type="text/plain"
        data-category="!performance">
    Please enable cookies to show this form.
</script>
```
But this is only called after changing the option. How can I show a message on page load when the script is disabled? Do I need to use some absolute element that later gets covered by the script?

**Comment by orestbida** [comment]
Do you want to show a message when no consent has been expressed at all (consent modal visible) or when a specific category has not been accepted?

**Comment by morja** [comment]
I think it would be great to be able to specify by category and by service. Pretty much like how it works now with `data-category="!performance"` or `data-service="!javascriptform"`.
